src/validators/OmnichannelConfigValidator.ts

```diff
@@ -7,9 +7,13 @@ const validateOmnichannelConfig = (omnichannelConfig: OmnichannelConfig): void =
       throw new Error(`OmnichannelConfiguration not found`);
     }
 
-    const currentOmnichannelConfigParams = Object.keys(omnichannelConfig);
+    const mapEntries = new Map(Object.entries(omnichannelConfig));
+    let valueOfEntry;
+
     for (const key of requiredOmnichannelConfigParams) {
-      if (!currentOmnichannelConfigParams.includes(key)) {
+      //check for they key present in the map, if not present or value is undefined then throw error
+      valueOfEntry = mapEntries.get(key);
+      if (!valueOfEntry || valueOfEntry.trim() === "") {
```

Comment:
I'd recommend to create a separate scenario for validation.

On `!valueOfEntry`, it checks where the key is present or not. If not present, it would throw `Missing '${key}' in OmnichannelConfiguration` as exception.

In the case for `valueOfEntry.trim() === ""`, I'd recommend to have a separate **if statement** to throw `'${key}' property is empty` as exception